CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+# Contributing to Containers/Image
+
+We'd love to have you join the community! Below summarizes the processes
+that we follow.
+
+## Topics
+
+* [Reporting Issues](#reporting-issues)
+* [Submitting Pull Requests](#submitting-pull-requests)
+* [Communications](#communications)
+<!--
+* [Becoming a Maintainer](#becoming-a-maintainer)
+-->
+
+## Reporting Issues
+
+Before reporting an issue, check our backlog of
+[open issues](https://github.com/containers/image/issues)
+to see if someone else has already reported it. If so, feel free to add
+your scenario, or additional information, to the discussion. Or simply
+"subscribe" to it to be notified when it is updated.
+
+If you find a new issue with the project we'd love to hear about it! The most
+important aspect of a bug report is that it includes enough information for
+us to reproduce it. So, please include as much detail as possible and try
+to remove the extra stuff that doesn't really relate to the issue itself.
+The easier it is for us to reproduce it, the faster it'll be fixed!
+
+Please don't include any private/sensitive information in your issue!
+
+## Submitting Pull Requests
+
+No Pull Request (PR) is too small! Typos, additional comments in the code,
+new testcases, bug fixes, new features, more documentation, ... it's all
+welcome!
+
+While bug fixes can first be identified via an "issue", that is not required.
+It's ok to just open up a PR with the fix, but make sure you include the same
+information you would have included in an issue - like how to reproduce it.
+
+PRs for new features should include some background on what use cases the
+new code is trying to address. When possible and when it makes sense, try to break-up
+larger PRs into smaller ones - it's easier to review smaller
+code changes. But only if those smaller ones make sense as stand-alone PRs.
+
+Regardless of the type of PR, all PRs should include:
+* well documented code changes
+* additional testcases. Ideally, they should fail w/o your code change applied
+* documentation changes
+
+Squash your commits into logical pieces of work that might want to be reviewed
+separate from the rest of the PRs. But, squashing down to just one commit is ok
+too since in the end the entire PR will be reviewed anyway. When in doubt,
+squash.
```

Comment:
(? I’m more the “when in doubt, split it” camp; for me, reviewing a hundred 10-line commits, as long as each commit is a logical, self-contained step, is much easier than reviewing a 1000-line PR which only outlines the broad goals.  I don’t care care about this _too_ strongly, but this seems surprising to me.  @runcom is this an intentional preference, or just something copy&pasted over 50 projects?)

Reply:
(OTOH some people tend create many-commit PRs where each commit is just a random collection of unrelated edits; those are useless, sure. But GitHub allows reviewing the PR as a whole, so even this does not hurt reviewing—only `git bisect`.)

Reply:
FWIW, Dan grabbed this doc (I think from skopeo) and dropped it into CRI-O with a few changes to point to the right project/issue links.  I've done the same for this one and a couple others.  I'm fine with stating whatever, but if we need to change this one, we ought to change the other 4 or 5 too.

But FWIW, I totally concur with the smaller is better commit.  kpod recently had a 5K line submit and that was a beast.

Reply:
> I'm fine with stating whatever, but if we need to change this one, we ought to change the other 4 or 5 too.

\*shrug\* I have a preference, but I’m not _quite_ as self-assured to insist that everything must be done that way :)

Let’s merge this, and I can submit a follow-up PR to change this bit.